alpha/experience-platform-extension/set-up-the-sdk.md

```diff
@@ -104,11 +192,14 @@ Depending on the nature of the event, and the various Adobe solutions you enable
 
 For the best performance, the server-side event handle comes in chunks. As a result, the `responseCallback` is called multiple times.
```

Comment:
Let's clarify as follows:
```
As a result, the `responseCallback` (Android) / `responseHandler` (iOS) is called multiple times.
```

Reply:
Same comment above: As described in the [Sending events](#sending-events) section...